doc/sphinx-guides/source/container/running/demo.rst

```diff
@@ -0,0 +1,132 @@
+
+Once Dataverse is running, you can check which version you have through the normal methods:
+
+- Check the bottom right in a web browser.
+- Check http://localhost:8080/api/info/version via API.
+
+About the Containers
+--------------------
+
+If you run ``docker ps``, you'll see that multiple containers are spun up in a demo or evaluation. Here are the most important ones:
+
+- dataverse
+- postgres
+- solr
+- smtp
+- bootstrap
+
+Most are self-explanatory, and correspond to components listed under :doc:`/installation/prerequisites` in the (traditional) Installation Guide, but "bootstrap" refers to :doc:`../configbaker-image`.
+
+Additional containers are used in development (see :doc:`../dev-usage`), but for the purposes of a demo or evaluation, fewer moving (sometimes pointy) parts are included.
+
+Security
+--------
+
+Please be aware that for now, the "dev" persona is used to bootstrap Dataverse, which means that admin APIs are wide open (to allow developers to test them; see :ref:`securing-your-installation` for more on API blocking), the "create user" key is set to a default value, etc. You can inspect the dev person `on GitHub <https://github.com/IQSS/dataverse/blob/master/modules/container-configbaker/scripts/bootstrap/dev/init.sh>`_ (look for ``--insecure``).
+
+We plan to ship a "demo" persona but it is not ready yet. See also :ref:`configbaker-personas`.
+
+Common Operations
+-----------------
+
+Starting the Containers
++++++++++++++++++++++++
+
+First, download :download:`compose.yml <../../../../../docker/compose/demo/compose.yml>` and place it somewhere you'll remember.
+
+Then, run ``docker compose up`` in the directory where you put ``compose.yml``
+
+Starting the containers for the first time involves a bootstrap process. You should see "have a nice day" output at the end.
+
+Stopping the Containers
++++++++++++++++++++++++
+
+You might want to stop the containers if you aren't using them. Hit ``Ctrl-c`` (hold down the ``Ctrl`` key and then hit the ``c`` key).
+
+You data is still intact and you can start the containers again with ``docker compose up``.
+
+Deleting the Containers
++++++++++++++++++++++++
+
+If you no longer need the containers because your demo or evaluation is finished and you want to reclaim disk space, run ``docker compose down`` in the directory where you put ``compose.yml``.
+
+Deleting the Data Directory
++++++++++++++++++++++++++++
+
+Data related to the Dataverse containers is placed in a directory called ``docker-dev-volumes`` next to the ``compose.yml`` file. If you are finished with your demo or evaluation or you want to start fresh, simply delete this directory.
```

Comment:
Should we continue to call the directory "docker-dev-volumes"? For a demo something like "dataverse-data" might make more sense.